.eslintrc.json

```
@@ -22,7 +22,7 @@
     ],
     "ignorePatterns": ["**/*.js", "coverage/**"],
     "rules": {
-      "no-console": "warn"
+      "no-console": "off"
```

```suggestion
      "no-console": "warn"
```

We've talked about a logging library. I'd be open to this if it were an override applied only to `infrastructure/` but for now, I'd like to have it keep warning at least in the app code as an indication of the fact there's still work to be done here.

---

OK that's fine. I'll create a ticket to standardize logging and then we can revisit this.